.github/workflows/release-winget.yml

```diff
@@ -0,0 +1,12 @@
+name: Publish to WinGet
+on:
+  release:
+    types: [released]
```

Comment:
does winget have a  prerelease channel too?

Reply:
Yes, that can be configured too.